app/tests/unit/controllers/submission.spec.ts

```diff
@@ -1244,3 +1244,138 @@ describe('updateSubmission', () => {
     expect(next).toHaveBeenCalledTimes(1);
   });
 });
+
+describe('assignPriority', () => {
```

Comment:
Values should reference defined enum types.

Reply:
This would mean that we're just checking the enum values against the enum values. We should use the values that the submission form uses cause if they some how get changed then this should break this.